docs/getting-started/kubernetes/crds.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: "Quickstart: Kubernetes Custom Resources"
+sidebar_label: Custom Resources
```

Comment:
Could you add a description to these new pages for improved SEO performance? Thanks